[ui] View ecosystem data

evamillan opened the pull request:
Creates a view at `/ecosystem/id` to visualize an ecosystem's metadata and the list of projects under it.
The data is fetched using the new `getEcosystemById` query.

Closes #67.

evamillan committed:
[ui] Create Ecosystem view

Creates a view at '/ecosystem/id' to visualize an ecosystem's
metadata and the list of projects under it.
The data is fetched using the 'getEcosystemById' query.

Signed-off-by: Eva Millán <evamillan@bitergia.com>

sduenas reviewed (approved):
LGTM

sduenas commented on ui/src/apollo/queries.js:
```diff
@@ -155,4 +191,10 @@ const getProjectByName = (apollo, name, ecosystemId) => {
+export {
+  getEcosystems,
+  getEcosystemByID,
+  GetBasicProjectInfo,
```
Just a minor comment, shouldn't this be `getBasicProjectInfo`?

evamillan replied:
You're right, I've renamed it.

evamillan committed:
[ui] Rename getBasicProjectInfo mutation

Changes the name from 'GetBasicProjectInfo' to
'getBasicProjectInfo' so it follows the same naming
convention as the rest of the mutations.

Signed-off-by: Eva Millán <evamillan@bitergia.com>

sduenas reviewed (approved):
LGTM. would you mind to rebase it?

evamillan commented:
Rebased.
